src/App.js

```diff
@@ -11,12 +12,14 @@ import Main from './Main'
 export default function App() {
   return (
     <AuthenticationProvider>
-      <SecretsProvider>
-        <PaperProvider theme={theme}>
-          <Main />
-          <StatusBar style="auto" />
-        </PaperProvider>
-      </SecretsProvider>
+      <RootSiblingParent>
```

Comment:
can you briefly explain this change or point to the docs which explain why it's necessary?

Reply:
Yes, when updating the expo it also has updated the clipboard package,  you can check this expo section https://docs.expo.dev/ui-programming/react-native-toast/#cross-platform--react-native-root-toast

Reply:
Makes sense, let's leave this comment here for reference. Looks good, thanks.